### PR TITLE
Add UI Component for UAA Auth

### PR DESCRIPTION
**Pull request by ninjadq**
It should show UAA auth configurations when select auth mode to UAA auth.

### CODE REVIEW

File: src/ui_ng/src/app/config/auth/config-auth.component.html [hunk 1]
```diff
@@ -15,6 +15,30 @@
             </a>
         </div>
     </section>
+
+    <section class="form-block" *ngIf="showUAA">
+        <div class="form-group">
+            <label for="uaa-endpoint" class="required">{{'CONFIG.UAA.ENDPOINT' | translate}}</label>
+            <input type="text" id="uaa-endpoint" name="uaa-endpoint" placeholder="endpoint" size="35"
```
Comment (steven-zou) [inline src/ui_ng/src/app/config/auth/config-auth.component.html]: Placeholder should be i18n text. The text might be uaa endpoint

File: src/ui_ng/src/app/config/auth/config-auth.component.html [hunk 2]
```diff
@@ -15,6 +15,30 @@
...
+            [(ngModel)]="currentConfig.uaa_endpoint.value" [disabled]="!currentConfig.uaa_verify_cert.editable">
+        </div>
+        <div class="form-group">
+            <label for="uaa-id" class="required">{{'CONFIG.UAA.CLIENT_ID' | translate}}</label>
+            <input type="text" id="uaa-cid" name="uaa-client-id" size="35"
+            [(ngModel)]="currentConfig.uaa_client_id.value" [disabled]="!currentConfig.uaa_verify_cert.editable">
```
Comment (steven-zou) [inline src/ui_ng/src/app/config/auth/config-auth.component.html]: If setting placeholder, set for all for consistence

Comment (ninjadq) [reply src/ui_ng/src/app/config/auth/config-auth.component.html]: I removed placeholders. UAA config has default value. And this input filed can not edit, placeholder may confuse the users.

Comment (jessehu) [reply src/ui_ng/src/app/config/auth/config-auth.component.html]: One minor comment: better to add a description for the 'uaa-id' field that the UAA CLIENT ID must have the following privilege: 

authorized-grant-types: password
scope: openid

@reasonerjt please correct me if anything missed.

**Pull request merged**